engine/runtime/src/main/scala/org/enso/interpreter/instrument/NotificationHandler.scala

```diff
@@ -26,6 +27,12 @@ trait NotificationHandler extends ProgressReporter with LockUserInterface {
     libraryVersion: LibraryVersion,
     location: Path
   ): Unit
+
+  /** A request to serialize the module.
+    *
+    * @param moduleName qualified module name
+    */
+  def serializeModule(moduleName: QualifiedName): Unit
```

Comment:
Adding a new method with every message is an incompatible change. Do we want to do it everytime we add new message? Don't we want to do something like `LogRecord` and just _"log a message"_ via a single `log` message?

Reply:
The notification handler is supposed to send `Runtime.Api` messages (to the language server) for specific events. There is no language server in text mode, so the methods are just no-ops.